components/ActivityIndicator/examples/ActivityIndicatorExample.m

```diff
@@ -35,6 +35,7 @@ - (id)init {
   self = [super init];
   if (self) {
     self.title = @"Activity Indicator";
+    self.colorScheme = [[MDCSemanticColorScheme alloc] init];
```

Comment:
I could be using outdated information, but I believe examples are supposed to provide `setColorScheme` and `setTypographyScheme` methods (or writable properties) in order to [automatically receive the theme from the catalog](https://github.com/material-components/material-components-ios/blob/develop/catalog/MDCCatalog/MDCNodeListViewController.swift#L634)?

Reply:
https://www.pivotaltracker.com/story/show/157064672 Default initializers serve a fallback role if the perforSelectors are not called, for example within the dragons app